docs/source/utils.rst

```diff
@@ -113,10 +113,78 @@ This detects ``address`` format and selects the socket family automatically.
     sock = bind_socket(address="::1", port=1234)
 
 
+Recurring callback
+++++++++++++++++++
+
+Runs coroutine function periodically with used defined strategy.
+
+.. code-block:: python
+
+    from typing import Union
+    from aiomisc import new_event_loop, RecurringCallback
+
+
+    async def callback():
+        print("Hello")
+
+
+    FIRST_CALL = False
+
+
+    async def strategy(_: RecurringCallback) -> Union[int, float]:
+        global FIRST_CALL
+        if not FIRST_CALL:
+            FIRST_CALL = True
+            # Delay 5 second if just started
+            return 5
+
+        # Delay 5 second if just started
+        return 10
+
+
+    if __name__ == '__main__':
+        loop = new_event_loop()
+
+        periodic = RecurringCallback(callback)
+
+        task = periodic.start(strategy)
+        loop.run_forever()
+
+
+The main purpose is this class is the strategy is an asynchronous function
+and you may write really flexible strategy.
```

Comment:
The main purpose is this class is to provide ability to specify the asynchronous strategy function, which can be written very flexible.